inference-engine/ie_bridges/java/org/intel/openvino/IECore.java

```diff
@@ -0,0 +1,95 @@
+package org.intel.openvino;
+
+import java.util.Map;
+
+public class IECore extends IEWrapper {
+    public static final String NATIVE_LIBRARY_NAME = "inference_engine_java_api";
+
+    public IECore() {
+        super(GetCore());
+    }
+
+    public IECore(String xmlConfigFile) {
+        super(GetCore_1(xmlConfigFile));
+    }
+
+    public CNNNetwork ReadNetwork(final String xml, final String bin) {
+        return new CNNNetwork(ReadNetwork1(nativeObj, xml, bin));
+    }
+
+    public CNNNetwork ReadNetwork(final String model) {
```

Comment:
can we rename `model` to `xml`? to avoid confusions. `model` cab be understood and string content of IR itself, but not as xml file name

Reply:
BTW, to avoid another kind of usage confusion:
- args with file names: use Java's "File" type (use `.getAbsolutePath()` internally)
- in-memory models: use `ByteArray` type or similar container

Reply:
BTW this method also allows to read native ONNX model

Reply:
> BTW this method also allows to read native ONNX model

Ok, let's use `modelFileName` then.